resources/views/pages/import.blade.php

```diff
@@ -6,6 +6,9 @@
 
     <div class="container -padded">
         <div class="wrapper">
+            <div class="container__block -narrow">
+                <p>For the time being, This feature should only be used by Jen Ng. If you have any questions please reach out to Jen directly or post a message in the #rogue channel in slack</p>
+            </div>
```

Comment:
Should there be a picture of Jen here?

Reply:
"Is this a picture of you? IF NOT, KEEP OUT."

Reply:
Hahahaha all of these are funny things and I fully support. 

But for the text that's there right now, lowercase `this` and a period after slack!